### PR TITLE
Fix typos in s3.s after manual review.

### PR DESCRIPTION
**Pull request by sebras**

### CODE REVIEW

File: src/sys/s3.s [hunk 1]
```diff
@@ -291,6 +291,10 @@ rkbdi:
    sad o134
    skp				" \| map to Ll ???
    jmp 2f
+   lacq
+   xor o40
```
Comment (philbudne) [inline src/sys/s3.s]: Looks like this swaps upper and lower case for all characters other than the ones mapped above.
The SimH simulation of the G2 device may need to be changed to reflect this as well!

Comment (sebras) [reply src/sys/s3.s]: > Looks like this swaps upper and lower case for all characters other than the ones mapped above.

Oh! I had no idea what the code did. I just spotted the difference.

> The SimH simulation of the G2 device may need to be changed to reflect this as well!

Aha, then let's make @larsbrinkhoff aware of this. Perhaps those changes need to go in at about the same time.

Comment (larsbrinkhoff) [reply src/sys/s3.s]: I'm now aware of this.

**Comment by teverett** [comment]
Confirmed

Screen Shot 2020-03-14 at 3 21 20 PM

**Comment by teverett** [comment]
@philbudne @sebras @larsbrinkhoff I see some comments.  ok to pull this?

**Comment by larsbrinkhoff** [comment]
Ok for me.  We'll update the simulator later, if needed.

**Comment by sebras** [comment]
@teverett Seems to me that @larsbrinkhoff got the heads up that was requested, so it's ok with me.

**Comment by sebras** [comment]
I rebased the branch on top of master to make the merge arrows in git log nicer. The changes are identical.

**Pull request merged**